extra-dev/packages/coq-hott/coq-hott.8.13.dev/opam

```diff
@@ -4,21 +4,29 @@ homepage: "http://homotopytypetheory.org/"
 bug-reports: "https://github.com/HoTT/HoTT/issues"
 license: "BSD-2-Clause"
 build: [
-  ["bash" "-c" "./autogen.sh -skip-submodules || :"]
-  ["./configure" "COQBIN=%{bin}%" "--prefix=%{prefix}%"]
-  [make "-j%{jobs}%"]
+  ["coq_makefile" "-f" "_CoqProject" "-o" "CoqMakefile"]
+  [make "-f" "CoqMakefile" "-j%{jobs}%"]
 ]
-install: [make "install"]
+install: [make "-f" "CoqMakefile" "install"]
 depends: [
-  "conf-autoconf" {build}
   "ocaml"
   "ocamlfind" {build}
   "coq" {>= "8.13" & < "8.14~"}
```

Comment:
```suggestion
  "coq" {= "dev"}
```
or even just "coq", I don't know what is the preferred way for .dev package (for sure not to require a non dev version)

Reply:
Ah, a misunderstanding on my side - after our discussion I assumed this is going to be a release package. But since the name is coq-hott.8.13.dev it is reasonable to tie it to Coq 8.13.